helm/cluster-operator-chart/templates/deployment.yaml

```diff
@@ -21,11 +23,9 @@ spec:
           preferredDuringSchedulingIgnoredDuringExecution:
           - podAffinityTerm:
               labelSelector:
-                matchExpressions:
```

Comment:
Just a question for @giantswarm/sig-operator, why we need podAnitiAffinity (or PodAffinity in general) if it would be 1 replica only?

Reply:
I agree that's weird.

Reply:
Ah this is `podAntiAffinity`. So I guess this is for sanity. Doesn't do much.